tool_path_planner/src/plane_slicer_raster_generator.cpp

```diff
@@ -124,8 +124,7 @@ static vtkSmartPointer<vtkPoints> applyParametricSpline(const vtkSmartPointer<vt
   new_points->InsertNextPoint(pt_prev.data());
 
   // adding remaining points by evaluating spline
-  double incr = point_spacing / total_length;
-  std::size_t num_points = std::ceil(total_length / point_spacing) + 1;
+  auto num_points = std::ceil(total_length / point_spacing) + 1;
```

Comment:
I'm not a fan of `auto`, but I know we don't have a strong organization style guide to point to one way or the other.  Personally, I think using the correct type will (over time, on average) lead to clearer and more easily understandable code.  If std::size_t is not the correct option here, I believe Eigen::Index would perform admirably in this case.

Reply:
I will admit to being a bit lazy here, but I just tried to fix some warnings about mixing signed and unsigned types. I personally don't mind using `auto` here because its fairly clear that the variable is an integer of some sort and the signedness doesn't really matter. I can update them to real types though